extensions/amp-story/1.0/page-advancement.js

```diff
@@ -719,6 +708,25 @@ export class ManualAdvancement extends AdvancementConfig {
     this.onTapNavigation(this.getTapDirection_(page));
   }
 
+  /**
+   * Calculates the pageRect based on the UIType.
+   * We can an use LayoutBox for mobile since the story page occupies entire screen.
+   * Desktop UI needs to use the higher latency getBoundingClientRect function.
```

Comment:
nit: I think `getBoundingClientRect` is actually "lower latency" than `LayoutBox`, since it's calculated more often? Unless I'm understanding something wrong... Maybe we can change the wording here?